INSTALL.md

```diff
@@ -1,10 +1,11 @@
 # Install Zync
-
-Download this repository and run `bundle install`.
-
-Install PostgreSQL with `brew install postgresql`
```

Comment:
Steps are numbered.
Instruction is in a line, and the command in another one:
 _3. Install PostgreSQL._
_`brew install postgresql`_